build.gradle

```diff
@@ -88,7 +89,7 @@ subprojects {
   }
 
   test {
-    maxHeapSize = "512m"
+    maxHeapSize = "1024m"
```

Comment:
ok - that's interesting, maybe that's a windows thing (our CI on cloudbees only tests on Linux)
That maybe worth asking @ljacomet about the need to have windows slaves.
Anyway, I'm ok with increasing to 1024m as long as there's a reason, and you just gave it - thank you !